products/cpdb/commitments/metadata.yml

```diff
@@ -0,0 +1,141 @@
+    data_type: text
+    example: 801HLDNSTJHN
+  - name: magency
+    display_name: magency
+    description: Three digit code of the distinct City agency managing the project.
+      The managing agency is the agency overseeing the construction or implementation
+      of a project.
+    data_type: integer
+    example: '846'
+  - name: projectid
+    display_name: projectid
+    description: Alphanumeric code created by the sponsor agency that identifies a distinct
+      project. A Project ID must be unique within a managing agency.
+    data_type: text
+    example: DOIT3MIFR
+  - name: projectdescription
+    display_name: projectdescription
+    description: Short description of the project as described by the sponsor agency.
+      If one projectid had many descriptions the longest description is reported by
+      CPDB.
+    data_type: text
+    example: JOB ORDER CONTRACTING WORK IN REGIONS NORTH, SOUTH & EAST
+
+  - name: sagencyacro
+    display_name: sagencyacro
+    description: Common acronym of the city agency sponsoring the planned commitment. This value is derived from the budget line.
+    data_type: text
+  - name: sagencyname
+    display_name: sagencyname
+    description: Common name for the city agency sponsoring the planned commitment. This value is derived from the budget line.
+    data_type: text
+
+  - name: budgetline
+    display_name: budgetline
+    description: Unique identifier of the budget used to fund the project.
+    data_type: text
+  - name: projecttype
+    display_name: projecttype
+    description: Short description of the type of project based on the budget funding the project.
+    data_type: text
+  - name: plancommdate
+    display_name: plancommdate
+    description: Month and year of when the planned commitment is projected to be committed.
+    data_type: datetime
+  - name: commitmentdescription
+    display_name: commitmentdescription
+    description: Short description of what the planned commtiment will specifically fund. Not all commitments have descriptions.
+    data_type: text
+  - name: commitmentcode
+    display_name: commitmentcode
+    description: Four letter or digit code indicating what part of the planned commitment the associated funding is for, such as design, construction, or contingency.
+    data_type: text
+  - name: typc
+    display_name: typc
+    description: Classification given by DCP based on keywords found in the short description
+      describing if a projects is Fixed Asset, Lump Sum, or ITT, Vehicles, and Equipment.
+    data_type: text
+    example: Fixed Asset
+  - name: typcname
+    display_name: typcname
+    description: Ten year plan category (typc) the planned commitment supports.
+    data_type: text
+
+  - name: plannedcommit_ccnonexempt
+    display_name: plannedcommit_ccnonexempt
+    description: Sum of City funding (Non-Exempt) associated with the planned commitment in the Capital Commitment Plan. 
+    data_type: integer
```

Comment:
Are these actually integers? (or is our validation for integers borked?)

Reply:
yup they're integers

![Screenshot 2024-06-20 at 11 27 42 AM](https://github.com/NYCPlanning/product-metadata/assets/7444289/b76594c3-e1fa-49a5-a34b-1273648ba5ca)